loops.js

```diff
@@ -51,3 +67,14 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++) {
+    if (i < 3) {
+        console.log(i);
+    }
+    if (i >= 3 && i < 6) {
```

Comment:
de if statement checkt hier of de conditie (in dit geval 'i') boven de 3 is. Maar omdat het hier boven al gecheckt is is het niet van noodzaak om het hier opnieuw te checken. 

Het is niet fout, maar scheelt jezelf wel weer code typen!

Reply:
ik snap niet wat je bedoelt. Mijn eerste if statement checkt toch juist of het kleiner is dan 3 en daarna van index nummer 3 tm 5?